apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/TroubleshootingDrawer.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Drawer, Flex, Text, Button, Link } from "@ledgerhq/react-ui";
+import { useHistory } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { ExternalLinkMedium } from "@ledgerhq/react-ui/assets/icons";
+
+import Animation from "~/renderer/animations";
+import { getDeviceAnimation } from "~/renderer/components/DeviceAction/animations";
+
+export type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const TroubleshootingDrawer = ({ isOpen, onClose }: Props) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+
+  return (
+    <Drawer big isOpen={isOpen} onClose={onClose}>
+      <Flex position="relative" flexDirection="column" height="100%" px={6}>
+        <Flex flexDirection="column" flex={1}>
+          <Animation
+            width="230px"
+            height="288px"
+            animation={getDeviceAnimation("nanoFTS", "light", "plugAndPinCode")}
```

Comment:
I think you can get the device model using a redux selector, and for the theme I think there's a way too